content/docs/gke/deploy/deploy-ui.md

```diff
@@ -6,35 +6,31 @@ weight = 3
 
 This page provides instructions for using the Kubeflow deployment web app to
 deploy Kubeflow on GCP. If you prefer to use the
-command line, see the guide to 
-[deployment using the CLI](/docs/gke/deploy/deploy-cli).
+command line, see [deployment using the CLI](/docs/gke/deploy/deploy-cli).
 
 1. Open [https://deploy.kubeflow.cloud/](https://deploy.kubeflow.cloud/#/deploy)
   in your web browser.
-1. Sign in using a GCP account that has administrator privileges for your 
-  GCP project.
-1. Complete the form, following the instructions on the left side of the form.
-  In particular, ensure that you enter the same **deployment name** as you used
-  when creating the OAuth client ID.
-1. Check **Skip IAP** box if you want to use basic auth.
-1. Click **Create Deployment**.
-
+2. Sign in using a GCP account that has administrator privileges for your GCP project. <!-- when i click on this link, there is no sign in page that gets loaded. Isn't this statement confusing for the user? -->
+3. Complete the form, following the instructions on the left side of the form. In particular, ensure that you enter the same deployment name as you used when creating the OAuth client ID. <!-- is OAuth client ID created before doing this? then we can link to that step here-->
 Here's a partial screenshot of the deployment UI, showing all the fields in the 
-form:
-
-<img src="/docs/images/kubeflow-deployment.png" 
+form: <img src="/docs/images/Kubeflow form.PNG" 
```

Comment:
Please restore the line break as in the original version. I'm not sure this image will render properly if it's on the same line as the text, even when the image is available.

Another thing to check: It looks like the image has a space in the name. That won't work, I don't think.

Also, when replacing an image rather than adding a new one, we should use the same image name. That way, if any other pages use the same image, they automatically get a refreshed image. Also, we avoid storing multiple images, some unused.

If you do want to change the image name, then you should delete the old image (after searching to see if it's used elsewhere in the docs).

Reply:
added line breaks, renamed images uploaded and all unwanted images removed